Source/WebCore/bindings/js/JSEventTargetCustom.h

```diff
@@ -34,23 +34,25 @@ namespace WebCore {
 
 // Wrapper type for JSEventTarget's castedThis because JSLocalDOMWindow and JSWorkerGlobalScope do not inherit JSEventTarget.
 class JSEventTargetWrapper {
-    WTF_MAKE_FAST_ALLOCATED;
 public:
+    JSEventTargetWrapper() = default;
+
     JSEventTargetWrapper(EventTarget& wrapped, JSC::JSObject& wrapper)
-        : m_wrapped(wrapped)
-        , m_wrapper(wrapper)
+        : m_wrapped(&wrapped)
+        , m_wrapper(&wrapper)
     { }
 
-    EventTarget& wrapped() { return m_wrapped; }
+    bool isValid() const { return m_wrapped; }
```

Comment:
We have competing idioms for this in WebKit. Sometimes `isValid`, sometimes `isNull`, in the standard library `has_value`, sometimes `explicit operator bool` and sometimes `operator!`. I wish it was more consistent.